assets/js/admin/job-tags-upsell.js

```diff
@@ -0,0 +1,25 @@
+const { registerPlugin } = wp.plugins
+const { PluginDocumentSettingPanel } = wp.editPost;
```

Comment:
Same question as above

```suggestion
import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
```

PS: We'll probably need to install `@wordpress/edit-post` in the project.

Reply:
Done in 606ef3afdda5b61c3027c6f52c1611a1f66959c5